src/components/Calendar.js

```diff
@@ -46,13 +48,21 @@ const Calendar = () => {
                 {d}
               </div>
             </div>
-            <div className='calendar-day-events'></div>
+            <div className='calendar-day-events'>
+              <ListGroup className='calendar-day-event-list'>
+                  { selectedMonthEvents[d] && selectedMonthEvents[d]
+                    .slice(0, 2)
+                    .map((ev, i) => <ListGroupItem key={i} className='calendar-day-event'>{ev.name}</ListGroupItem>) }
+                </ListGroup>
+                { selectedMonthEvents[d] && selectedMonthEvents[d].length > 2 &&
+                  <div className='calendar-day-more-events'> +{selectedMonthEvents[d].length - 2} </div>}
+            </div>
```

Comment:
Evenimentele din lista latesc coloana din calendar atunci cand lungimea numelui evenimentului e prea mare. Ai putea renunta la componentele `ListGroup` si `ListGroupItem` in favoarea unor `div`-uri stilizate de tine.